sound/soc/intel/boards/kbl_rt5660.c

```diff
@@ -138,13 +138,13 @@ static int kabylake_ssp0_fixup(struct snd_soc_pcm_runtime *rtd,
 {
 	struct snd_interval *rate = hw_param_interval(params,
 			SNDRV_PCM_HW_PARAM_RATE);
-	struct snd_interval *channels = hw_param_interval(params,
+	struct snd_interval *chan = hw_param_interval(params,
```

Comment:
in fact IMHO even file-static variables with such generic names like `channels` or `rates` aren't a very good thing. One reason is the problem, that we get here, another reason is, that if many files (in this case board drivers) have variables with the same name, searching for them using grep or (vi or emacs) tags becomes difficult. Ideally I'd rename those variables to be file-specific (e.g. `kabylake_rt5660_channels` just like `kabylake_rt5660_ops` that's already used there), but that's more work of course...